packages/suite/src/utils/suite/guide.ts

```diff
@@ -0,0 +1,38 @@
+import { GuideCategoryType, GuideItemType } from '@suite-types/guide';
+
+export const getItemTitle = (item: GuideItemType, language: string): string =>
+    item.title[language] || item.title.en;
```

Comment:
nit:
```suggestion
    item.title[language || 'en'];
```

OR

```
export const getItemTitle = (item: GuideItemType, language: string = 'en'): string =>
    item.title[language];
```

Reply:
I think this will not work. 

If e.g. czech title is not available, then I would like to switch to English title. Your suggestion solves problem when language is undefined.

Reply:
actually, my suggestions doesn't solve any problem, because there is no - that is why it's marked as a nitpick, I just wanted to point out other possibility to write it down in more condensed (and more readable from my point of view) syntax... if you don't like it, feel free to ignore it ;-)